packages/components/src/wrap.test.tsx

```diff
@@ -9,14 +8,16 @@ describe('wrap', () => {
 	);
 	Button.displayName = 'Button';
 	Button.foo = 'bar';
-	it('should create a component', () => {
+	it('should create a component', async () => {
+		const user = userEvent.setup();
```

Comment:
fix lint issue:
Property 'setup' does not exist on type '{ click: (element: Element, init?: MouseEventInit | undefined, { skipHover, clickCount, skipPointerEventsCheck, }?: (clickOptions & PointerOptions) | undefined) => void; ... 10 more ...; keyboard: { ...; }; }'.ts(2339)

Reply:
Same thing than for preventScroll.test.tsx